src/header.rs

```diff
@@ -843,7 +843,6 @@ impl<W: io::Write> UncompressedHeader for BitWriter<W, BigEndian> {
     if fi.large_scale_tile {
       unimplemented!();
     }
-    self.write_bit(true)?; // trailing bit
```

Comment:
Why is this removed?

Reply:
Because it's no longer needed. That bit is the trailing bit at the end of a Frame Header to signal the end of the entire OBU, and we don't write those anymore. Frame OBUs are composed by a Frame Header followed immediately by a Tile Group merged into a single OBU.

See section 5.10 in the spec.